utils/releaser/src/ReleaseWorker/Release/SetMutualDependenciesToVersionReleaseWorker.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker\Release;
+
+use PharIo\Version\Version;
+use Shopsys\Releaser\ReleaseWorker\AbstractSetMutualDependenciesToVersionReleaseWorker;
+use Shopsys\Releaser\Stage;
+
+final class SetMutualDependenciesToVersionReleaseWorker extends AbstractSetMutualDependenciesToVersionReleaseWorker
```

Comment:
this class should be added to `utils/releaser/config/release.php` file

Reply:
I do not know where it went